frontend/website-redesign/src/pages/GetStarted.re

```diff
@@ -0,0 +1,96 @@
+module Styles = {
+  open Css;
+
+  let background =
+    style([
+      backgroundImage(
+        `url("/static/img/BecomeAGenesisMemberBackground.png"),
+      ),
+      backgroundSize(`cover),
+    ]);
+};
+
+[@react.component]
+let make = () => {
+  <Page title="Mina Cryptocurrency Protocol" footerColor=Theme.Colors.orange>
+    <div className=Nav.Styles.spacer />
+    <Hero
+      title="Get Started"
+      header="Mina makes it simple."
+      copy={
+        Some(
+          "Interested and ready to take the next step? You're in the right place.",
+        )
+      }
+      background={
+        Theme.desktop: "/static/img/backgrounds/04_GetStarted_1_2880x1504.jpg",
+        Theme.tablet: "/static/img/backgrounds/04_GetStarted_1_1536x1504_tablet.jpg",
+        Theme.mobile: "/static/img/backgrounds/04_GetStarted_1_750x1056_mobile.jpg",
+      }
+    />
+    <ButtonBar
+      kind=ButtonBar.GetStarted
+      backgroundImg="/static/img/ButtonBarBackground.png"
+    />
+    <AlternatingRowsButton
```

Comment:
why was this renamed to append `button`?

Reply:
the buttons were different than the link, maybe we could have the component accept children instead?

Reply:
i made a new component bc it was a bit different, didn't have the top numbers either

Reply:
sounds good! was just wondering!